ftw/zipextract/implementations_at.py

```diff
@@ -15,7 +15,7 @@
 
 def normalize_id(name):
     normalizer = component.getUtility(IIDNormalizer)
-    normalized = normalizer.normalize(name)
+    normalized = normalizer.normalize(name).lstrip('_')
```

Comment:
Now I just want to know what happens with a file called `test.txt` and another called `_test.txt`, or even better a directory called `test` and a file called `_test` ??!!

Reply:
It goes to the namechooser and then that one will generate something like `test-1` etc.

Reply:
https://github.com/4teamwork/ftw.zipextract/blob/90ee96cce9c8f645471b710e78486bf07061c446/ftw/zipextract/implementations_at.py#L32-L34